### ISSUE TITLE
API: Compatibility with LNDHub/BlueWallet

### ISSUE THREAD

**Issue by dennisreimann** [dennisreimann/btcpayserver#4]
An API compatible with [LNDHub](https://github.com/BlueWallet/LndHub/blob/master/doc/Send-requirements.md) would potentialy allow to use LNbank with BlueWallet.

**Comment by bumi** [comment]
and it could also be connected to Alby  \o/

**Comment by prusnak** [comment]
@dennisreimann Two questions:
1. Is there any progress on this?
2. Is there any mobile wallet that can work with the native LNBank API?

**Comment by dennisreimann** [comment]
Yes, I've got a branch for that already and will try to finish it. Recently finished the LNDhub integration into our Lightning lib, hope I'll get to this one soon. 

To my knowledge there is no other native mobile app that works with LNbank right now.

**Comment by iBobik** [comment]
No, it will be the same custodial wallet like that default in BlueWallet. Just very convenient compared to LN bank UI.

Honza Pobořil
💬Signal

**Comment by dennisreimann** [comment]
@ishristov As @iBobik said, this issue tracks the compatibility wof LNbank with the BlueWallet app: My goal is to make the LNbank wallets available inside BlueWallet, so that they can be used with a nice mobile interface too.

The other feature you mentioned got merged just yesterday. In btcpayserver/BTCPayServer.Lightning#74 I implemented LNDhub (which backs the regular BlueWallet accounts) as a Lightning node option for BTCPay Server. With the next release it will be possible to have a LNDhub account/wallet as Lightning node for a store.
